eng/pipelines/libraries/stress/http.yml

```diff
@@ -84,42 +95,49 @@ jobs:
     clean: true
     fetchDepth: 5
     lfs: false
-
   - powershell: |
       $(dockerfilesFolder)/build-docker-sdk.ps1 -w -t $(sdkBaseImage) -c $(BUILD_CONFIGURATION)
       echo "##vso[task.setvariable variable=succeeded;isOutput=true]true"
     name: buildRuntime
     displayName: Build CLR and Libraries
-
```

Comment:
It's probably better to keep these line breaks isn't it?

(See https://github.com/dotnet/runtime/pull/58860#discussion_r712247737)

Reply:
Removed all whitespace changes.